vision/google/cloud/vision/image.py

```diff
@@ -85,28 +85,58 @@ def source(self):
         """
         return self._source
 
-    def _detect_annotation(self, feature):
+    def _detect_annotation(self, features):
```

Comment:
I can't find any place in this PR where you've passed in a list with anything other than one element. Is the change to `features` not YAGNI?